catalogue/wrds/reference-data-and-schema-lookup/spec/open-api.yml

```diff
@@ -0,0 +1,88 @@
+openapi: 3.0.1
+info:
+  title: WRDS - Reference Data & Schema Lookup
+  description: "_Retrieves table schemas and data listing all fields in the database._\n\nA RESTful service providing similar functionality to the standard lookup API for retrieving reference data from WRDS.\nAdditionally with this API you can retrieve the table schema to build POCO objects for deserialising the responses.\nThis is available nationally.\n"
```

Comment:
The title and description will be overridden by the catalogue processor.  Ok to leave the title in here as it's a required field but I suggest leaving out the description to avoid confusion about where it should be placed
```suggestion
```